gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -0,0 +1,375 @@
+import hail as hl
```

Comment:
Note: Part way through this file I stopped changing `release` to `version`, can you change them all?